### DISCUSSION TITLE
--config-file pointing to a non-existent file

### DISCUSSION THREAD

**Discussion by GeorgeMitrakis** [cypress-io/cypress#9291] [Questions and Help] [answered]
Hi all !

This is my first post here. I hope everyone is healthy and safe.

I have noticed something regarding the `--config-file` cli option.
If I set to it a non-existent file as a value, it creates it with an empty json object. For example:
`$ npx cypress open --config-file nonexistent.json`
creates the nonexistent.json file with `{ }` in it, if it doesn't exist.

When something like this: `'./nonexistent.json'` is passed as a value, a `'.` directory is created, and under it, a `nonexistent.json'` file. 
The  `'.` directory cannot be deleted directly from the filesystem (Windows).

Is this the desired behavior?
I find it a little weird.

Thanks in advance.

**Comment by jennifer-shehane** [comment, marked as answer]
No, this is not the intended behavior, sounds like a bug. It should error if the config file is not found, we have code around this. https://github.com/cypress-io/cypress/blob/develop/packages/server/lib/errors.js#L677:L677

It may be something specific to windows. We'd be open to an issue/PR to fix this.
